### PR TITLE
Use this.options for subgenerator and remove fountainPrompting() call

### PR DESCRIPTION
**Pull request by zckrs**

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -137,19 +133,19 @@ module.exports = fountain.Base.extend({
...
 
-      if (this.props.js === 'babel') {
+      if (this.options.js === 'babel') {
         this.mergeJson('package.json', {devDependencies: {'babel-preset-es2015': '^6.2.0'}});
       }
     }
   },
 
   writing: {
     wireing() {
```
Comment (micaelmbagira) [inline generators/app/index.js]: Little typo here :)

Comment (zckrs) [inline generators/app/index.js]: done replaced by "wiring"

**Comment by micaelmbagira** [comment]
👍

**Comment by Swiip** [comment]
👍

**Pull request merged**